eng/ignore-links.txt

```diff
@@ -28,3 +28,4 @@ https://docs.microsoft.com/javascript/api/@azure-rest/arm-compute?view=azure-nod
 https://docs.microsoft.com/javascript/api/@azure-rest/arm-network?view=azure-node-preview
 https://docs.microsoft.com/javascript/api/@azure/arm-securitydevops?view=azure-node-preview
 https://docs.microsoft.com/javascript/api/@azure/arm-devhub?view=azure-node-preview
+https://docs.microsoft.com/javascript/api/@azure/load-testing
```

Comment:
same here, should this be https://docs.microsoft.com/javascript/api/@azure-rest/load-testing

Reply:
Updated.